### DISCUSSION TITLE
Showing off:  What I build with FluentUI

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#17673] [Show and tell]
Hi all,

These are some screenshots of an application that I am developing using `FluentUI`, `TypeScript`, `NodeJs` and the almighty `PostgreSQL`. The back-end of this app is a collection of generated REST APIs based on the database schema and some other `JSON` schema definitions. The front-end is of course all built with `FluentUI`. Some components like the `ApplicationLauncher` are custom components modelled after `Office365`. For legacy reason I had to implement a basic user management module, but soon this will be replaced with `Azure AD`. As for state management; I use `MobX` for global application state and `useState` for tracking component state locally.

This application helps tracking SDGs (Sustainable Development Goals) for large organizations. _(read about SDGs [here](https://sdgs.un.org/goals))_

Thanks for your time.

![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/b44310bf10844bf91f7ad4a5b8a03e64a96a0429/0.png)

![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/b44310bf10844bf91f7ad4a5b8a03e64a96a0429/1.png)

![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/b44310bf10844bf91f7ad4a5b8a03e64a96a0429/2.png)

![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/b44310bf10844bf91f7ad4a5b8a03e64a96a0429/4.png)

![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/b44310bf10844bf91f7ad4a5b8a03e64a96a0429/5.png)

![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/b44310bf10844bf91f7ad4a5b8a03e64a96a0429/6.png)

![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/b44310bf10844bf91f7ad4a5b8a03e64a96a0429/7.png)

![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/b44310bf10844bf91f7ad4a5b8a03e64a96a0429/3.png)

**Comment by khmakoto** [comment]
@blendsdk This is awesome! Thanks for sharing!

**Reply by JustSlone** [reply]
+1 on what Mak said. Thanks for sharing!
